docs/src/main/paradox/server-side/graceful-termination.md

```diff
@@ -1,5 +1,33 @@
 # Graceful termination
 
+## Akka Coordinated Shutdown
+
+Coordinated shutdown is Akka's managed way of shutting down multiple modules / sub-systems (persistence, cluster, http etc)
+in a predictable and ordered fashion. For example, in a typical Akka application you will want to stop accepting new HTTP connections, and then shut down the cluster etc.
+
+Akka HTTP shutdown consists mainly of three steps:
+
+1. stop accepting new connections (@scala[@scaladoc[ServerBinding.unbind](akka.http.scaladsl.Http.ServerBinding)]@java[@javadoc[ServerBinding.unbind](akka.http.javadsl.ServerBinding)])
+1. close open connections (@scala[@scaladoc[ServerBinding.terminate](akka.http.scaladsl.Http.ServerBinding)]@java[@javadoc[ServerBinding.terminate](akka.http.javadsl.ServerBinding)])
+1. shut down connection pools (@scala[@scaladoc[Http.shutdownAllConnectionPools](akka.http.scaladsl.HttpExt)]@java[@javadoc[Http.shutdownAllConnectionPools](akka.http.javadsl.Http)])
```

Comment:
Why is this necessary at all after unbinding and terminating?

Reply:
the first 2 items are about the server side, `shutdownAllConnectionPools` is about the client side.